docs/Contributing.adoc

```diff
@@ -1,17 +1,40 @@
-== Contributing 
+== Contributing
 
 If you're reading this, hopefully you are considering helping out with the Ansible Agnostic Deployer for Red Hat enablement.
 
 Herein lies the contribution guidelines for helping out with this project. Do take the guidelines here literally, if you find issue with any of them or you see room for improvement, please let us know via a GitHub issue.
 
-=== Rules
+=== General rules
 
 * The Ansible <<coc,Code of Conduct>> still applies.
 * For git messages, branch names, ..., follow <<gitstyle,Git Style Guide>>
 * Should you wish to work on a completely new standard, GREAT, but please file an issue for tracking.
 * To contribute, fork and make a pull request against the `development` branch.
 * All tasks should be in YAML literal.
 
+=== About reviewing Pull Requests
+
+. Do not merge a PR after reviewing unless explicitely asked for
+** Approval and merging are different.
+** Other people might be reviewing at the same time, or want to review that PR
+** The PR might not be fully tested by the author yet
```

Comment:
I think rule "If your PR is not ready mark it as so" is quite enough. If author forgot, he should say so after merge and it can get fixed or reverted. If it's fast, it's not as big of a deal :)

Reply:
What about guidance for author how to mark PRs (and pieces of code) that need advice/help on? :)

Reply:
GitHUb supports draft PRs so why do we not use that + a standard WIP in the title? https://github.blog/2019-02-14-introducing-draft-pull-requests/